From 1aa0bd90d904e614d8bd274cda515d8d352ac967 Mon Sep 17 00:00:00 2001
From: Max Ihlenfeldt <max@igalia.com>
Date: Wed, 7 Feb 2024 11:58:30 +0000
Subject: [PATCH] Don't require profiler_builtins.rlib

This library is only needed for profiling builds, which we don't do.
Including it leads to compile errors on with OE's current master branch
because they disabled building the library, so just skip it.

Upstream-Status: Inappropriate [specific to our build setup]
Signed-off-by: Max Ihlenfeldt <max@igalia.com>
---
 build/rust/std/BUILD.gn | 1 -
 1 file changed, 1 deletion(-)

diff --git a/build/rust/std/BUILD.gn b/build/rust/std/BUILD.gn
index fdddf68..7afe61a 100644
--- a/build/rust/std/BUILD.gn
+++ b/build/rust/std/BUILD.gn
@@ -76,7 +76,6 @@ if (toolchain_has_rust) {
   # don't need to pass to the C++ linker because they're used for specialized
   # purposes.
   skip_stdlib_files = [
-    "profiler_builtins",
     "rustc_std_workspace_alloc",
     "rustc_std_workspace_core",
     "rustc_std_workspace_std",
